### ISSUE TITLE
Using a non-tuple sequence for multidimensional indexing is deprecated; use `arr[tuple(seq)]` instead of `arr[seq]`

### ISSUE THREAD

**Issue by nbanik** [jobovy/galpy#347]
Hi Jo, 

   This future warning popped up after I updated the packages : 

/home/nil/anaconda3/lib/python3.6/site-packages/galpy/potential/SCFPotential.py:857: FutureWarning: Using a non-tuple sequence for multidimensional indexing is deprecated; use `arr[tuple(seq)]` instead of `arr[seq]`. In the future this will be interpreted as an array index, `arr[np.array(seq)]`, which will result either in an error or a different result.
  s+= nu.prod(wp[index])*integrand(*xp[index])

Thanks,
Nil

**Comment by nbanik** [comment]
Hi Jo, 

   I am getting the same error but now for different lines. 

WARNING: Using particle spray technique, effects of progenitor's orbit incorporated
C:\Users\nbani\Anaconda3\lib\site-packages\galpy\potential\SCFPotential.py:438: FutureWarning: Using a non-tuple sequence for multidimensional indexing is deprecated; use `arr[tuple(seq)]` instead of `arr[seq]`. In the future this will be interpreted as an array index, `arr[np.array(seq)]`, which will result either in an error or a different result.
  self._computeforce(R[j][0],z[j][0],phi[j][0])

C:\Users\nbani\Anaconda3\lib\site-packages\galpy\potential\SCFPotential.py:439: FutureWarning: Using a non-tuple sequence for multidimensional indexing is deprecated; use `arr[tuple(seq)]` instead of `arr[seq]`. In the future this will be interpreted as an array index, `arr[np.array(seq)]`, which will result either in an error or a different result.
  force[j] = dr_dx[j][0]*dPhi_dr + dtheta_dx[j][0]*dPhi_dtheta +dPhi_dphi*dphi_dx[j][0]

**Comment by jobovy** [comment]
I'm surprised you get this warning, because I thought I test whether this warning is raised. Could you try:
```
import warnings
with warnings.catch_warnings(record=True):
    warnings.simplefilter("error",FutureWarning)
    p= mockSCFZeeuwPotential()
    p.Rforce(1.,0.)
```
because that is the test that is run. Either this fails for you, and then maybe it's a version issue (you are using v1.4 right?).

**Comment by nbanik** [comment]
How do I import mockSCFZeeuwPotential ? I didn't run the tests after installing galpy, yes I am running v1.4 (used conda forge which should install the latest version? )

**Comment by jobovy** [comment]
Ah, good point! It's defined in ``tests/test_potential.py``, but I think you can just do
```
import numpy
from galpy import potential
from galpy.util import bovy_coords
def rho_Zeeuw(R, z=0., phi=0., a=1.):
    r, theta, phi = bovy_coords.cyl_to_spher(R,z, phi)
    return 3./(4*numpy.pi) * numpy.power((a + r),-4.) * a
##Mock SCF class                                                         
class mockSCFZeeuwPotential(potential.SCFPotential):
    def __init__(self):
        Acos, Asin = potential.scf_compute_coeffs_spherical(rho_Zeeuw,2)
        potential.SCFPotential.__init__(self,amp=1.,Acos=Acos, Asin=Asin)

**Comment by nbanik** [comment]
I just ran this along with the test in the previous comment and there was no output. I should mention that when I construct the bar, I don't get any warning message, but when I include it in the spraydf function and sample, then I get this message. This message didn't appear when I was running galpy on linux. In fact, I am running the bar+spiral notebook which ran fine on linux.

**Comment by jobovy** [comment]
I think I figured out the issue: the issue was for array input, so things like
```
p.Rforce(numpy.atleast_1d(1.),numpy.atleast_1d(0.))
```
I've pushed a fix for this, but I guess you're stuck on v1.4 so can't test this?

**Comment by nbanik** [comment]
I went directly to SCFpotential.py and made the change and now it's working fine. Thanks!
